docs/source/installation.mdx

```diff
@@ -23,10 +23,12 @@ pip install bitsandbytes
 
 ```bash
 git clone https://github.com/TimDettmers/bitsandbytes.git && cd bitsandbytes/
-cmake -B build -DBUILD_CUDA=ON -S .
+pip install -r requirements-dev.txt
+cmake -DCOMPUTE_BACKEND=cuda
```

Comment:
```suggestion
cmake -DCOMPUTE_BACKEND=cuda -S .
```
Do we also need this here?

Reply:
@younesbelkada 
> Do we also need this here?

No harm in it. It's optional though; works the same for me with and without it.

Reply:
ok perfect, thanks @matthewdouglas for confirming !